generic-desktop-kde-bsc1105798-20180824.json

```diff
@@ -0,0 +1,31 @@
+{
+  "properties": [
+    "workaround",

```

Comment:
I guess single property is enough =) Please remove and we can merge

Reply:
But, this way you are sure ;)

Reply:
but seriously, how did you manage to get 4 times the tag into here? I guess you did not do this manually but over the webUI, I guess this is an openQA bug then?

Reply:
Haha, strange. Of course, I did that via webUI.